core/chaincode/lifecycle/lifecycle.go

```diff
@@ -436,6 +436,8 @@ func (ef *ExternalFunctions) ApproveChaincodeDefinitionForOrg(chname, ccname str
 		return errors.WithMessagef(err, "could not set defaults for chaincode definition in channel %s", chname)
 	}
 
+	privateName := fmt.Sprintf("%s#%d", ccname, requestedSequence)
```

Comment:
Did you forget to remove this?

Reply:
Nope, it's being used in lines 462 and 475